app/simulation_test.go

```diff
@@ -315,8 +309,12 @@ func TestAppImportExport(t *testing.T) {
 
 	ctxA := bApp.NewContextLegacy(true, tmproto.Header{Height: bApp.LastBlockHeight()})
 	ctxB := newApp.NewContextLegacy(true, tmproto.Header{Height: bApp.LastBlockHeight()})
-	newApp.ModuleManager().InitGenesis(ctxB, bApp.AppCodec(), genesisState)
-	newApp.StoreConsensusParams(ctxB, exported.ConsensusParams)
+	if _, err = newApp.ModuleManager().InitGenesis(ctxB, bApp.AppCodec(), genesisState); err != nil {
+		panic(err)
+	}
+	if err = newApp.StoreConsensusParams(ctxB, exported.ConsensusParams); err != nil {
+		panic(err)
+	}
```

Comment:
why panic here and not use `require.NoError(t, ...)`?

```suggestion
	_, err = newApp.ModuleManager().InitGenesis(ctxB, bApp.AppCodec(), genesisState)
	require.NoError(t, err, "failed to initialize genesis")
	err = newApp.StoreConsensusParams(ctxB, exported.ConsensusParams)
	require.NoError(t, err, "failed to store consensus params")
```